src/Strategy/CommonClassesStrategy.php

```diff
@@ -82,13 +83,25 @@
                 'condition' => [\Buzz\Client\FileGetContents::class, \Buzz\Message\ResponseBuilder::class],
             ],
         ],
+        Psr18Client::class => [],
     ];
 
     /**
      * {@inheritdoc}
      */
     public static function getCandidates($type)
     {
+        if (Psr18Client::class === $type) {
```

Comment:
add a comment that we merge the httplug clients that implement psr18 too?

Reply:
Sure